hw2/05-form.html

```diff
@@ -10,10 +10,57 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="bg-primary container mt-5">
+    <main class=" bg-light rounded p-4 d-flex flex-column max-sm-75">
       <form>
-        <h1>Exercise 05 - Form</h1>
+        <div class="mb-3">
+          <label for="fullName" class="form-label">Full name</label>
```

Comment:
Label needs to be in bold, can use the class `fw-bold` to do this. Labels on line 21, 25, 34, and 55 also need to be bolded.